OpenRA.Mods.Common/Traits/SelfHealing.cs

```diff
@@ -9,19 +9,30 @@
  */
 #endregion
 
+using System.Collections.Generic;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Attach this to actors which should be able to regenerate their health points.")]
 	class SelfHealingInfo : UpgradableTraitInfo, Requires<HealthInfo>
 	{
+		[Desc("Absolute amount of health points added in each step.")]
 		public readonly int Step = 5;
+
+		[Desc("Relative percentages of health added in each step.")]
```

Comment:
In the description, can you mention the fact that the sum of `Step` and `PercentageStep` gets applied when both are defined, please?